vowpalwabbit/global_data.cc

```diff
@@ -85,6 +85,18 @@ int print_tag_by_ref(std::stringstream& ss, const v_array<char>& tag)
 
 int print_tag(std::stringstream& ss, v_array<char> tag) { return print_tag_by_ref(ss, tag); }
 
+std::string vw::get_setup_name(reduction_setup_fn setup_fn)
```

Comment:
can both these functions have better names?

Reply:
Do have any suggestions for better names?

Reply:
nit: for `map_setup_name` maybe something like `build_setupfn_name_dict`? for `get_setup_name` maybe just adding setupfn i.e. `get_setupfn_name`

Reply:
Changed.